CalculatorApp/Calculator.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalculatorApp
+{
+    public class Calculator
+    {
+        public int add(string numbers)
+        {
+            int sum = 0;
+            if (string.IsNullOrEmpty(numbers) || numbers.Length == 0)
+            {
+                return sum;
+            }
+            bool IsValidInput = ValidateInput(numbers);
+            if (IsValidInput)
+            {
+                string[] splittedInput = DataSplit(numbers);
+                if (splittedInput.Length >=1)
+                {
+                    int[] arrayOfNumbers = InputParsing(splittedInput);
+                    foreach (int number in arrayOfNumbers)
+                    {
+                        sum += number;
+                    }
+                }
+                else
+                {
+                    throw new ArgumentException("Invalid Input!");
+                }
+            }
+            else
+            {
+                throw new ArgumentException("Invalid Input!");
+            }
+
+            return sum;
+        }
+        private string[] DataSplit(string input)
+        {
+            string[] splitedInput = input.Split(',');
+            return splitedInput;
+        }
+
+        private int[] InputParsing(string[] numbers)
+        {
+            int[] parsedNumbers = new int[numbers.Length];
```

Comment:
hmmm, lets try to use lists instead of arrays
also, don't forget LINQ u learned ^_*